code/game/objects/items/stacks/nanopaste.dm

```diff
@@ -32,6 +32,28 @@
 		else
 			to_chat(user, "<span class='notice'>[src] won't work on that.</span>")
 
+/obj/item/stack/nanopaste/afterattack(atom/A, mob/user, proximity_flag)
+	if(!ismecha(A) || user.a_intent == INTENT_HARM || !proximity_flag)
+		return
+	var/obj/mecha/mecha = A
+	if((mecha.obj_integrity >= mecha.max_integrity) && !mecha.internal_damage)
+		to_chat(user, "<span class='notice'>[mecha] is at full integrity!</span>")
+		return
+	if(mecha.state == MECHA_MAINT_OFF)
+		to_chat(user, "<span class='warning'>[mecha] cannot be repaired without maintenance protocols active!</span>")
+		return
+	if(mecha.repairing)
+		to_chat(user, "<span class='notice'>[mecha] is currently being repaired!</span>")
+		return
+	if(mecha.internal_damage & MECHA_INT_TANK_BREACH)
+		mecha.clearInternalDamage(MECHA_INT_TANK_BREACH)
+		user.visible_message("<span class='notice'>[user] repairs the damaged gas tank.</span>", "<span class='notice'>You repair the damaged gas tank.</span>")
```

Comment:
```suggestion
		user.visible_message("<span class='notice'>[user] repairs the damaged air tank.</span>", "<span class='notice'>You repair the damaged air tank.</span>")
```
I would change this to air instead of gas, since it is related to breathing.